scripting/pugsetup.sp

```diff
@@ -12,7 +12,7 @@
 #include <SteamWorks>
 
 #undef REQUIRE_PLUGIN
-#include "include/updater.inc"
+#include <updater>
```

Comment:
This change will break the automated compilation test  since this file won't be available: https://github.com/splewis/csgo-pug-setup/blob/master/.travis.yml

I would revert this, unless there's a reliable location to get the include from (i.e., undo https://github.com/splewis/csgo-pug-setup/commit/ca7688571a2e95b05bfa6164b5f09886cee8cae5)